readme.md

```diff
@@ -14,11 +14,11 @@
 
 ##### Installation
 
-The `R` package is presently under development and therefore not currently listed on [`CRAN`](https://cran.r-project.org/). For the meantime, please download the `R` package file associated with the [latest release](https://github.com/psych-ds/validator/releases/latest).
-
-In `RStudio`, please then select `Install Packages ...` from the `Tools` menu. Next, please choose `Package Archive File` in the `Install from` dropdown, and select the file you downloaded in the last step. After confirming your selection, the installation should proceed automatically.
-
-Alternatively, you can install the package from the `R` command line with the command `install.packages("~/path/to/download/psychds_x.y.z.tar.gz", repos=NULL, type="source")`.
+The `R` package is presently under development and therefore not currently listed on [`CRAN`](https://cran.r-project.org/). In the meantime, you can install the latest version of the package from the R command line with the commands:
```

Comment:
I'm not sure -- Do you think we should deprecate the `RStudio` installation mechanism?

Reply:
Good point. Let's not. Will just fix the download-latest-version link instead.

Reply:
👍 Awesome, thank you!